audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
Wait wait wait! There are 3 more voice lines for Soviet Engineer, we can use one of them. http://www.ppmforums.com/viewtopic.php?highlight=engineer&t=33239